osidb/renderers.py

```diff
@@ -25,7 +25,6 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         if data is None:
             data = {}
         data["dt"] = timezone.now()
-        data["revision"] = os.getenv("OPENSHIFT_BUILD_COMMIT") or "unknown"
```

Comment:
@JimFuller-RedHat Is it safe to remove this, or is there any other reason for having it?

Reply:
@jobselko we do not want to remove this. Have a look at https://osidb.prodsec.redhat.com/osidb/api/v1/flaws/CVE-2023-36053?include_fields=x

Reply:
Thanks, I thought it would have some meaning :)